content/c-sharp/concepts/math-functions/terms/min/min.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: '.Min()'
+Description: 'Returns the smaller of two specified numbers.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Numbers'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+# .Min()
+
+The **`Math.Min()`** method returns the lesser of two specified numbers.
+
+## Syntax
+
+```pseudo
+Math.Min(num1, num2)
+```
+
+`Math.Min()` returns the lesser value from `num1` and `num2`.
+
+## Example
+
+The following example compares two integers `i1` and `i2` and writes the lesser integer to the console.
+
+```cs
+using System;
+
+public class Example {
+  public static void Main (string[] args) {
+    int i1 = -134, i2 = 268;
+
+    Console.WriteLine("Math.Min (int) = " + Math.Min(i1,i2));
+  }
+}
+```
+
+This example results in the following output:
+
+```shell
+Math.Min (int) = -134
+```
+
+Now it's your turn, try changing the values of xint1 and xint2 to see the results.
```

Comment:
```suggestion
## Codebyte Example

The following code is runnable and the values of xint1 and xint2 can be changed to see the results.
```

We generally avoid first and second person pronouns, and the codebyte example should have a header.